tagger/api.py

```diff
@@ -78,20 +78,37 @@ def endpoint_interrogate(self, req: models.TaggerInterrogateRequest):
         if req.image is None:
             raise HTTPException(404, 'Image not found')
 
+        if isinstance(req.model, list):
```

Comment:
This is to allow combining (averaging) the output of multiple interrogators, like in the ui.